renderer/ShaderProgram.h

```diff
@@ -0,0 +1,217 @@
+/*
+* Descent 3
+* Copyright (C) 2024 Parallax Software
```

Comment:
Change it to "Copyright (C) 2024 Descent Developers" since this is new addition to code.